Classes/WeaponFire_Link.uc

```diff
@@ -3,7 +3,7 @@ class WeaponFire_Link extends LinkFire;
 event ModeDoFire()
 {
     if(!LinkGun(Weapon).Linking)

```

Comment:
.Linking is referring to shafting your teammate to form a link chain. Knowing that, this makes sense because we don't want to count that towards the fired count / accuracy. More info on what I found here: https://github.com/mich-devs/3SPNRU/issues/10